pkgs/tools/admin/olivetin/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
```

Comment:
Would you be okay with rewriting this to build from source using `buildGoModule`? Right now we're only able to build for 64-bit linux, but the go compiler supports building for both 64 bit and 32 bit linux and darwin.